examples/io/split-jpeg-to-fits.py

```diff
@@ -34,8 +34,9 @@
 
 image = Image.open('Hs-2009-14-a-web.jpg')
 xsize, ysize = image.size
-print("Image size: {} x {}".format(xsize, ysize))
-plt.imshow(image)
+print("Image size: {} x {}".format(ysize, xsize))
+print("Image bands: {}".format(image.getbands()))
```

Comment:
```suggestion
print(f"Image size: {ysize} x {xsize}")
print(f"Image bands: {image.getbands()}")
```
These might look cleaner in 3.6-style (especially the first); also it would be a good opportunity to introduce people to f-strings.

Reply:
I don't see this suggestion applied even though the comment in https://github.com/astropy/astropy/pull/10364#issuecomment-636183849 implies it was. FYI.

Reply:
Did you click on "accept change" (or something like this) @ADiegoCAlonso?

Reply:
I was using both github UI and the command line to edit the commit. As expected, it didn't work well. Apologies for the confusion

Reply:
You'd probably need to fetch the updates from here before proceeding to work on the local clone, though I am surprised it did not require a `push --force` after different changes in both locations were made...